pkgs/development/compilers/gcc/builder.sh

```diff
@@ -201,11 +201,11 @@ postConfigure() {
 
 
 preInstall() {
+    mkdir -p "$out/${targetConfig}/lib"
```

Comment:
Still want `[ -z "$enableMultilib" ]` though, right?

Reply:
Those directories should always be created. I think the logic for the conditional below is correct (other than `is64bit` referring to wrong platform). On 32-bit platforms, there should only be a `lib` directory. On 64-bit platforms, there should be `lib` and `lib64`, which should be linked unless multilib is enabled.

It doesn't seem to break anything to have `lib64` on 32-bit as well (since it happens unintentionally in the armv7l cross compiler), but that kind of change seems out of scope for this PR.

Reply:
Well, I'm not sure we ever don't do `enableMultilib`, but I think with multi-lib it never prefixes any directories.

Reply:
Multilib is disabled for normal builds. In all the 64-bit GCCs I have built, `lib64` is a symlink to `lib`. The only place I can find in nixpkgs where `enableMultilib` is true is `gcc_multi`.

Reply:
Oh oops I forgot the `-z` and got it backwards.